readthedocs/templates/core/send_email_form.html

```diff
@@ -12,14 +12,14 @@
   </p>
 
   <ul>
-    {% for user in users|slice:":20" %}
-      <li>{{ user.email }} ({{ user }})</li>
+    {% for recipient in recipients|slice:":20" %}
```

Comment:
I'm rather impartial. This isn't important logic, and is for display purposes only, so seems like template logic should be fine.